### DISCUSSION TITLE
Add Custom Icons for commands?

### DISCUSSION THREAD

**Discussion by Syrkres** [phibr0/obsidian-commander#85] [General]
Is there a way (or can you point me in a direction) for adding custom icons for the commands? I really like the plugin but would like to add some custom icons.

**Comment by michae1ho11ey** [comment]
I too really wanted to be able to add icons to this plugin and so I thought I'd just dive into the source and do it myself! Well I learned a few of things diving into the code:

1. I can't add my own icons to the plugin, because the plugin doesn't host them.
2. The plugins are pulled from Lucide Icons (https://github.com/lucide-icons/lucide   _I think this is the right repo_.
3. If you have any artistic skill, you can add your icons here and eventually it'll be added to Commander (assuming they enable every new icon with every build of commander).
4. By default, when picking an icon you can only scroll through a small subset of all possible icons (A-Ca). If you instead use the search bar at the top of the picker then you can select any from the whole list 500+ icons, if you know the right name.
5. There really isn't an easy place to see all of the icons. To see the full list and what they look like you'll need to clickon every svg file in the repo: https://github.com/lucide-icons/lucide/tree/main/icons

Not quite what you wanted, but at least you now know more about how the icons work.

**Reply by StewPedassle** [reply]
For posterity, Lucide icons can be searched at https://lucide.dev/icons/
